tests/sentry/integrations/slack/service/test_slack_service.py

```diff
@@ -93,6 +92,7 @@ def setUp(self) -> None:
 
     def test_none_group(self):
         self.activity.update(group=None)
+        self.service._logger = mock.MagicMock()
```

Comment:
mypy was helping you here, monkeypatching a class like this leaks outside of the scope of the test

this should use a context manager so it is undone after the test finishes:

```python
with mock.patch.object(self.service, '_logger') as self.mock_logger:
    ...
```

Reply:
i get the same mypy error when i add the context manager

Reply:
can you show the mypy error you're referring to?